lib/questions.rb

```diff
@@ -213,12 +255,13 @@ def count_words_of_each_length_in_a_file(file_path)
 # implement fizzbuzz without modulo, i.e. the % method
 # go from 1 to 100
 # (there's no RSpec test for this one)
-def fizzbuzz_without_modulo
+def fizzbuzz_without_modulo(value)
```

Comment:
Unused method argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used. You can also write as `fizzbuzz_without_modulo(*)` if you want the method to accept any arguments but don't care about them.